ROADMAP.md

```diff
@@ -69,25 +60,18 @@ These are goals achievable within 6-12 months, tentatively done by mid 2023.
   When most of the API can be used asynchronously.
 
 
-- **Beta availability of the extension in k6 Cloud for all users.**<br>
-
-  *How will we achieve this?*<br>
-  The deployment should be optimized and the extension thoroughly tested before making it available to all users. Frontend changes should be done at this point, and usage costs (CPU, RAM, storage) and pricing details should be determined, followed by public announcements of the availability. Features such as screen capture, video recording, downloading, and file uploading should be available.
-
-  *Definition of Done*<br>
-  When all users are eligible to upgrade to a plan that includes browser testing.
+Mid-term goals
+--------------
 
+These are goals tentatively achievable by Q1 2024.
 
-- **Merge the extension into the main k6 repository.**<br>
-  k6 and Grafana consider browser testing to be another strategy under the testing umbrella, along with performance, contract, and other types of testing that modern web applications benefit from. As such, the scope of the k6 tool will expand to include functional/E2E testing using browser automation, with the ultimate goal of merging the xk6-browser extension into the main k6 repository as a core JS module.
+- **Global availability of the extension in k6 Cloud for all users.**<br>
 
   *How will we achieve this?*<br>
-  Once the extension and API are relatively stable, and it's well-tested in k6 Cloud, we will create a PR to the main k6 repository with the extension code. Then we will deprecate the standalone xk6-browser repository, and point users to use the main k6 binary instead.
-
-  This will be a minor breaking change, as scripts will have to import `k6/browser` instead of `k6/x/browser`, and CI jobs will need to change, but the functionality should remain the same.
+  The deployment should be optimized and the extension thoroughly tested before making it available to all users. Frontend changes should be done at this point, and usage costs (CPU, RAM, storage) and pricing details should be determined, followed by public announcements of the availability.
 
   *Definition of Done*<br>
-  Once the extension has been merged into the k6 repository.
+  The import path for xk6-browser is updated to `k6/browser` and the API is considered to be stable.
 
 
 - **Increase test code coverage; refactor problematic areas of the codebase; fix "flaky" tests, linter issues, etc.**<br>
```

Comment:
I believe this should be under the short-term goals. Or, maybe, an all-encompassing goal. Like reserving some percent of the time in each milestone to stabilize the k6-browser product.

Reply:
Ahha, got it.